app/cdap/components/PipelineHistory/PipelineHistoryTableRow.tsx

```diff
@@ -33,6 +33,7 @@ interface IPipelineHistoryTableRowProps {
 }
 
 const PREFIX = 'features.PipelineHistory.table';
+const SNAPSHOTVERSION = '-SNAPSHOT';
```

Comment:
What do you think adding this to `global-constants.js`?

Reply:
sounds good! I've added it to global-constants